src/App.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+let todoList = [
+  { id: 1, title: "Homework" },
+  { id: 2, title: "Exercise" },
+  { id: 3, title: "Study" },
+];
+
+let list = todoList.map((item, index) => (
+  <li key={index}>
+    {" "}
+    {item.id} : {item.title}{" "}
+  </li>
+));
```

Comment:
Good job, here! 💯  The `key` should be `item.id`, though. It's kind of frowned upon to use the loop's `index` as the key in react.

Reply:
Thank you for the feedback, I will fix it for next time. 🙂